[2024-06-03T22:24:40.308+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T22:24:40.325+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-06-03T22:24:40.334+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-06-03T22:24:40.335+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T22:24:40.347+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-12-01 00:00:00+00:00
[2024-06-03T22:24:40.355+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=30343) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T22:24:40.357+0000] {standard_task_runner.py:63} INFO - Started process 30345 to run task
[2024-06-03T22:24:40.358+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-12-01T00:00:00+00:00', '--job-id', '135', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp2168f0m2']
[2024-06-03T22:24:40.359+0000] {standard_task_runner.py:91} INFO - Job 135: Subtask blunders_etl
[2024-06-03T22:24:40.399+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-12-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T22:24:40.480+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-01T00:00:00+00:00'
[2024-06-03T22:24:40.481+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T22:24:40.482+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T22:24:40.491+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T22:34:59.349+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T22:34:59.350+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T22:34:59.359+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-12-01T00:00:00+00:00, execution_date=20211201T000000, start_date=20240603T222440, end_date=20240603T223459
[2024-06-03T22:34:59.385+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T22:34:59.396+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T22:34:59.397+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T19:50:17.561+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T19:50:17.575+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-06-08T19:50:17.581+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-06-08T19:50:17.581+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T19:50:17.590+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-12-01 00:00:00+00:00
[2024-06-08T19:50:17.597+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=30796) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T19:50:17.599+0000] {standard_task_runner.py:63} INFO - Started process 30798 to run task
[2024-06-08T19:50:17.598+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-12-01T00:00:00+00:00', '--job-id', '311', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpuyfp3bv8']
[2024-06-08T19:50:17.599+0000] {standard_task_runner.py:91} INFO - Job 311: Subtask blunders_etl
[2024-06-08T19:50:17.632+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-12-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T19:50:17.692+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-01T00:00:00+00:00'
[2024-06-08T19:50:17.694+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T19:50:17.694+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T19:50:17.700+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T20:00:16.082+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T20:00:16.082+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T20:00:16.092+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-12-01T00:00:00+00:00, execution_date=20211201T000000, start_date=20240608T195017, end_date=20240608T200016
[2024-06-08T20:00:16.111+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T20:00:16.129+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T20:00:16.131+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-08T01:48:41.311+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T01:48:41.325+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-07-08T01:48:41.331+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-12-01T00:00:00+00:00 [queued]>
[2024-07-08T01:48:41.331+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-08T01:48:41.340+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-12-01 00:00:00+00:00
[2024-07-08T01:48:41.348+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=31675) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T01:48:41.349+0000] {standard_task_runner.py:63} INFO - Started process 31677 to run task
[2024-07-08T01:48:41.349+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-12-01T00:00:00+00:00', '--job-id', '697', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp9vtqba54']
[2024-07-08T01:48:41.349+0000] {standard_task_runner.py:91} INFO - Job 697: Subtask blunders_etl
[2024-07-08T01:48:41.381+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-12-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-08T01:48:41.447+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-01T00:00:00+00:00'
[2024-07-08T01:48:41.448+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T01:48:41.449+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-08T01:48:41.455+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-08T01:58:27.533+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-08T01:58:27.534+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T01:58:27.540+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-12-01T00:00:00+00:00, execution_date=20211201T000000, start_date=20240708T014841, end_date=20240708T015827
[2024-07-08T01:58:27.574+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T01:58:27.590+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T01:58:27.592+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
